Subject: ui: details: Increased scrolledview6's height request
From: nose-gnome 71675376+nose-gnome@users.noreply.github.com Thu Mar 23 16:07:29 2023 +0000
Date: Sat Mar 2 14:19:28 2024 -0500:
Git: bb86721352a3478614ea69503f12e61cddaeee18

Increased the height of scrolledview6 to make viewing applications  easier in os info

diff --git a/ui/details.ui b/ui/details.ui
index 29ed119b..ae68ea67 100644
--- a/ui/details.ui
+++ b/ui/details.ui
@@ -788,7 +788,7 @@
                             <property name="margin-top">3</property>
                             <child>
                               <object class="GtkScrolledWindow" id="scrolledwindow6">
-                                <property name="height-request">150</property>
+                                <property name="height-request">200</property>
                                 <property name="visible">True</property>
                                 <property name="can-focus">True</property>
                                 <property name="hexpand">True</property>
